data/prod/persian/articles/cyddjz5058wo.json

```diff
@@ -33,7 +33,8 @@
           "thingUri": "http://www.bbc.co.uk/things/c6176220-b6bb-4a1c-b3b4-3a6fcdc283b1#id",
           "thingId": "c6176220-b6bb-4a1c-b3b4-3a6fcdc283b1",
           "thingLabel": " پهپاد",
-          "thingType": ["Thing", "Theme"]
+          "thingType": ["Thing", "Theme"],
+          "thingSameAs": []
```

Comment:
@jimjohnsonrollings could you confirm that Ares should be supplying `thingSameAs` values whenever we can a valid `about` object?

Reply:
The implementation seems to be that `thingSameAs` is always there, but it is allowed to be empty.  If we'd rather it just went away rather than being empty, we can ask for that.

Reply:
Its being empty is fine, I'm mainly asking will the field always be returned from Ares?

Reply:
Yes